runtime/oti/j9nonbuilder.h

```diff
@@ -4769,6 +4769,7 @@ typedef struct J9InternalVMFunctions {
 	U_32 ( *getVMRuntimeState)(struct J9JavaVM *vm);
 	BOOLEAN ( *updateVMRuntimeState)(struct J9JavaVM *vm, U_32 newState);
 	U_32 ( *getVMMinIdleWaitTime)(struct J9JavaVM *vm);
+	void ( *registerCPEntry)(struct J9JavaVM *javaVM, J9ClassPathEntry *cpEntry);
```

Comment:
Also could we just move this to the end of the struct rather than in the middle?